recipes/tests.rb

```diff
@@ -90,23 +90,29 @@
 # SGE
 ###################
 if node['cfncluster']['cfn_scheduler'] == 'sge'
+  sge_bin_suffix = if arm_instance?
+                     "arm64"
+                   else
+                     "amd64"
+                   end
+  sge_bin_paths = "/opt/sge/bin:/opt/sge/bin/lx-#{sge_bin_suffix}"
```

Comment:
not related to this line, but in the sge_install recipe there is a guard on `creates '/opt/sge/bin/lx-amd64/sge_qmaster'`. I think that should be managed as well otherwise the installation will be repeated at runtime too

Reply:
good catch i very much missed that one.